CodeStarterCamp_Week2/Step2.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Step2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by Daehoon Lee on 2023/03/01.
+//
+
+import Foundation
+
+func chooseLottoNumbers() -> Set<Int> {
+    var pickUpLottoNumbers: Set<Int> = Set<Int>()
+    
+    while pickUpLottoNumbers.count < 6 {
+        pickUpLottoNumbers.insert(Int.random(in: 1...45))
+    }
+    
+    return pickUpLottoNumbers
+}
+
+func compareLottoNumbers(myLottoNumbers: [Int], chosenLottoNumbers: Set<Int>) -> [Int] {
+    return chosenLottoNumbers.intersection(myLottoNumbers).sorted()
+}
+
+func printResult(commonLottoNumbers: [Int]) {
+    let commonLottoNumbersString: [String] = commonLottoNumbers.map{ String($0) }
```

Comment:
1. 상수명에 String을 붙이는게 맞는가에 대해서 고민을 해보시면 좋을 것 같습니다 😀
[API Design Guidelines-Promote Clear Usage](https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage)을 보면
`Name variables, parameters, and associated types according to their roles, rather than their type constraints.`라 설명하고 있습니다.

또한, 모든 상수들은 각각의 타입을 가질텐데, 다른 상수 이름에는 타입이 없네요! 일관성이 없고, 사실상 지키기도 어렵죠 😀 

2. 여기만 컨벤션이 조금 다르네요 😀 
`map`과 `{` 사이에 공백이 필요해보입니다 😀
아주 사소해 보이지만 공백 하나, 개행 하나에도 이유가 있어야합니다 😎

Reply:
1. `commonLottoNumbersString` 상수를 지우고 설명해주신 체이닝을 통해 해결하였습니다.
  ```Swift
  commonLottoNumbers.map { String($0) }.joined(separator: ", ")
  ``` 
2. 아직 고차함수 map을 배우지 않은 상태에서 사용하다보니 공백이 필요하다는 것을 알지 못했습니다.😭